rules/aws/collection_cloudtrail_logging_created.toml

```diff
@@ -28,7 +28,7 @@ references = [
 risk_score = 21
 rule_id = "594e0cbf-86cc-45aa-9ff7-ff27db27d3ed"
 severity = "low"
-tags = ["AWS", "Elastic", "SecOps", "Logging", "Continuous Monitoring"]
+tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Log Auditing"]
```

Comment:
it looks like there's supposed to be a meaning to the ordering? if so, it's not obvious to a user.
kinda goes against the idea of tags, but I won't rehash that argument.

I have no idea how to convey any ordering with tags, and I just see this as a bag of phrases

Reply:
The ordering applied is just to satisfy my own desire for visual cleanliness and order between the rules.  I do not intend to use ordering for any programatic or UI function.  If you or anyone desires another order that opinion would be equally valid.  I do not even propose we test order for user submitted rules.